CONTRIBUTING.md

```diff
@@ -48,26 +48,25 @@ Now, run these commands in your preferred terminal in a good directory for devel
 ```
 git clone https://github.com/signalapp/Signal-Desktop.git
 cd Signal-Desktop
-npm install -g yarn      # (only if you don't already have yarn)
-npm install -g grunt-cli # (only if you don't already have grunt)
-yarn install             # Install and build dependencies (this will take a while)
-grunt                    # Generate final js/css assets
-yarn icon-gen            # Generate full set of icons for Electron
-yarn test                # A good idea to make sure tests run first
-yarn run start           # Run!
+npm install --global yarn      # (only if you don’t already have yarn)
+yarn install --frozen-lockfile # Install and build dependencies (this will take a while)
+yarn grunt                     # Generate final JS and CSS assets
```

Comment:
So, you're against direct use of `grunt`, eh? :0)

Reply:
I just prefer local over global dependencies where possible 😉This way people are less likely to run into problems due to mismatching `grunt` versions.

Reply:
I get that concern. But you should know that the way grunt works is that `grunt-cli` always defers to the locally-installed grunt anyway. :0)

Reply:
Thanks, I didn’t know that.

Reply:
One benefit with this change is that people don’t even have to install a global `grunt-cli` if they don’t want to.